internal/keycloakb/configdbmodule_test.go

```diff
@@ -8,25 +8,29 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/cloudtrust/common-service/database/sqltypes"
+	"github.com/cloudtrust/common-service/log"
 	"github.com/cloudtrust/keycloak-bridge/internal/dto"
 	"github.com/cloudtrust/keycloak-bridge/internal/keycloakb/mock"
+	msg "github.com/cloudtrust/keycloak-bridge/internal/messages"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 )
 
-func useMockDB(t *testing.T, fn func(DBConfiguration)) {
+func useMockDB(t *testing.T, fn func(sqltypes.CloudtrustDB)) {
```

Comment:
This function is not used... it can be removed

Reply:
thanks